### ISSUE TITLE
Class `kedro.extras.datasets.plotly.PlotlyDataSet` not found.

### ISSUE THREAD

**Issue by ahoo1260** [quantumblacklabs/kedro#845]
## Description
I tried to create a Plotly dataset in catalog.yml :

`cluster_plot:
    type: kedro.extras.datasets.plotly.PlotlyDataSet
    filepath: data/08_reporting/cluster_plot.json
    plotly_args:
        type: bar
        fig:
            x: features
            y: importance
            orientation: 'h'
        layout:
            xaxis_title: 'x'
            yaxis_title: 'y'
            title: 'Test'
`
and I received this error:

```
kedro.io.core.DataSetError: An exception occurred when parsing config for DataSet `cluster_plot`:
Class `kedro.extras.datasets.plotly.PlotlyDataSet` not found.
```

## Your Environment
* Kedro version : 0.17.3
* Python version :  3.7.10
* Operating system and version: Windows 10

**Comment by datajoely** [comment]
@ahoo1260 this is a silly error we're in the process of fixing - we are suppressing a `ModuleNotFound` error. Please make sure you have all the right requirements installed.

**Comment by lorenabalan** [comment]
Hi @ahoo1260 the PlotlyDataSet was only released in 0.17.4, so make sure you upgrade your kedro version to be able to use this dataset.
